Android/Google.Android.DataTransport/build.cake

```diff
@@ -0,0 +1,125 @@
+#tool nuget:?package=XamarinComponent
+
+#addin nuget:?package=Cake.XCode
+#addin nuget:?package=Cake.Xamarin.Build
+#addin nuget:?package=Cake.Xamarin
+#addin nuget:?package=Cake.FileHelpers
+
+var TARGET = Argument ("t", Argument ("target", "Default"));
+
+string JAR_URL = "";
+Dictionary<string, string> URLS_ARTIFACT_FILES= new Dictionary<string, string>()
+{
+	{
+		$"https://maven.google.com//com/google/android/datatransport/transport-api/2.0.0/transport-api-2.0.0.aar",
+		$"./externals/android/transport-api-2.0.0.aar"
```

Comment:
I would store these versions in variables so that we can XmlPoke the .csproj to update the files from here. Also, to prevent all the changes to the .csproj each version update, maybe leave off the version number from the downloaded file so that the .csproj just needs to reference the same "transport-api.aar".